lib/screens/app/wallet/dashboard.dart

```diff
@@ -450,4 +456,36 @@ class _DashboardState extends State<Dashboard> {
       ]),
     );
   }
+
+  Widget buildAppBar(BuildContext _context) {
+    return AppBar(
+      titleSpacing: 0,
+      leading: Container(
+        margin: const EdgeInsets.all(16),
+        child: SvgPicture.asset(
+          'assets/images/wallet/app_bar/appbar_logo.svg',
+        ),
+      ),
+      title: SvgPicture.asset('assets/images/wallet/app_bar/seeds_text.svg'),
+      actions: [
+        Container(
+          child: IconButton(
+            icon: SvgPicture.asset(
+              'assets/images/wallet/app_bar/scan_qr_code_logo_2.svg',
+              height: 30,
```

Comment:
no width needed here?

Reply:
could change the width even to 2000 that it will not affect the size